pkgs/by-name/an/angle/package.nix

```diff
@@ -73,6 +92,9 @@ stdenv.mkDerivation (finalAttrs: {
     "use_custom_libcxx=true"
     "angle_enable_swiftshader=false"
     "angle_enable_wgpu=false"
+    # On darwin during linking:
+    # clang++: error: argument unused during compilation: '-stdlib=libc++'
+    "treat_warnings_as_errors=false"
```

Comment:
Would `-Wno-error=unused-command-line-argument` work instead?

Or at least only turning this on for darwin?

I guess warnings arent a huge deal

Reply:
This would be the most common method I'd say

`env.NIX_CFLAGS_COMPILE = "-Wno-unused-command-line-argument";`

Maybe best to just leave it with gn options anyway. I don't mind :slightly_smiling_face: